Documentation/ceph-cluster-crd.md

```diff
@@ -131,18 +130,18 @@ Below are the settings available, both at the cluster and individual node level,
 
 - `useAllDevices`: `true` or `false`, indicating whether all devices found on nodes in the cluster should be automatically consumed by OSDs. **Not recommended** unless you have a very controlled environment where you will not risk formatting of devices with existing data. When `true`, all devices will be used except those with partitions created or a local filesystem. Is overridden by `deviceFilter` if specified.
 - `deviceFilter`: A regular expression that allows selection of devices to be consumed by OSDs.  If individual devices have been specified for a node then this filter will be ignored.  This field uses [golang regular expression syntax](https://golang.org/pkg/regexp/syntax/). For example:
-  - `sdb`: Only selects the `sdb` device if found
-  - `^sd.`: Selects all devices starting with `sd`
-  - `^sd[a-d]`: Selects devices starting with `sda`, `sdb`, `sdc`, and `sdd` if found
-  - `^s`: Selects all devices that start with `s`
-  - `^[^r]`: Selects all devices that do *not* start with `r`
+  - `sdb`: Only selects the `sdb` device if found.
+  - `^sd.`: Selects all devices starting with `sd`.
+  - `^sd[a-d]`: Selects devices starting with `sda`, `sdb`, `sdc`, and `sdd` if found.
+  - `^s`: Selects all devices that start with `s`.
+  - `^[^r]`: Selects all devices that do *not* start with `r`.
 - `devices`: A list of individual device names belonging to this node to include in the storage cluster.
   - `name`: The name of the device (e.g., `sda`).
   - `config`: Device-specific config settings. See the [config settings](#osd-configuration-settings) below.
 - `directories`:  A list of directory paths that will be included in the storage cluster. Note that using two directories on the same physical device can cause a negative performance impact.
   - `path`: The path on disk of the directory (e.g., `/rook/storage-dir`).
   - `config`: Directory-specific config settings. See the [config settings](#osd-configuration-settings) below.
-- `location`: Location information about the cluster to help with data placement, such as region or data center.  This is directly fed into the underlying Ceph CRUSH map. The type of this field is `string`. For example, to add datacenter location information, set this field to `rack=rack1`.  More information on CRUSH maps can be found in the [ceph docs](http://docs.ceph.com/docs/master/rados/operations/crush-map/).
+- `location`: Location information about the cluster to help with data placement, such as region or data center.  This is directly fed into the underlying Ceph CRUSH map. The type of this field is `string`. For example, to add data center location information, set this field to `rack=rack1`.  More information on CRUSH maps can be found in the [ceph docs](http://docs.ceph.com/docs/master/rados/operations/crush-map/).
```

Comment:
The trend I see in most of the rest of the docs and the guideline I see most often is to make sure bulleted list items do *not* end in a period.